.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
 - '8'
 - '10'
-- '11'
+- '12'
```

Comment:
has node 11 been removed on purpose here?

Reply:
node 11 is a short term release, is there any reason we should continue to support it?

Reply:
Not sure really. Probably not, if it's easy to upgrade from node 11 to node 12

Are we using node 10, 11, 12 anywhere? I thought all of the Life projects were using node 8

Reply:
10 has been fixed on Lambda.  I don't know of anyone using 11 or 12.  Since this is an open source project it didn't make sense to me to continue building with a deprecated version.

Reply:
Removing `11` in favor of `12` seems fine, but is unrelated to the rest of this PR.

Reply:
I can open a separate PR for it if necessary, I just figured I would change it since I was already making changed.

Reply:
Meh, it's already here now and is pretty small. Just generally nice to keep things separate.